### PR TITLE
Inquiry register message

### PR DESCRIPTION
**Pull request by keshavgr8**
Fixes bug [#154702995](https://www.pivotaltracker.com/story/show/154702995)

### CODE REVIEW

File: client-src/src/providers/helper/helper.ts [hunk 1]
```diff
@@ -28,4 +29,27 @@ export class HelperProvider {
...
+      for(var i = 0; i < ionItems.length; i++){
+        if (ionItems[i].classList.contains('ng-untouched')) {
+          ionItems[i].classList.remove('ng-untouched');
+          ionItems[i].classList.add('ng-touched');
+        }
+      }
+    }
+    let pinAutoComplete = document.getElementsByTagName("ng2-completer")[0];
+    if(pinAutoComplete){
+      if (pinAutoComplete.classList.contains('ng-untouched')) {
```
Comment (aayushKhandpur) [inline client-src/src/providers/helper/helper.ts]: Should merge the 2 conditions

**Review by aayushKhandpur — CHANGES_REQUESTED** [review]
Address review comments

File: client-src/src/providers/helper/helper.ts (hunk 1 above)
Comment (keshavgr8) [reply client-src/src/providers/helper/helper.ts]: The following conditions?
if(pinAutoComplete){
        if (pinAutoComplete.classList.contains('ng-untouched'))

Comment (aayushKhandpur) [reply client-src/src/providers/helper/helper.ts]: yes

Comment (keshavgr8) [reply client-src/src/providers/helper/helper.ts]: Addressed in commit 1ed18e1

**Review by aayushKhandpur — APPROVED** [review]

**Pull request merged**